components/script/dom/webglshader.rs

```diff
@@ -40,11 +41,20 @@ pub struct WebGLShader {
     renderer: WebGLMsgSender,
 }
 
-#[cfg(not(target_os = "android"))]
-const SHADER_OUTPUT_FORMAT: Output = Output::Glsl;
+#[cfg(not(any(target_os = "android", target_os = "ios")))]
+const SHADER_OUTPUT_FORMAT_WEBGL1: Output = Output::Glsl;
+// MacOS Requires Glsl150Core because it uses a OpenGL 3.2 Core Context.
```

Comment:
Instead of a bunch of per-OS hacks here, we should probably be looking at SHADING_LANGUAGE_VERSION

Reply:
Great idea! I'll update the PR to use SHADING_LANGUAGE_VERSION